robot/src/main/java/us/ilite/robot/modules/Limelight.java

```diff
@@ -0,0 +1,250 @@
+package us.ilite.robot.modules;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import us.ilite.common.config.SystemSettings;
+
+public class Limelight extends Module {
+
+    private final NetworkTable mTable = NetworkTableInstance.getDefault().getTable("limelight");
+    public TargetingData mCurrentTarget = null;
+
+    @Override
+    public void modeInit(double pNow) {
+
+    }
+
+    @Override
+    public void periodicInput(double pNow) {
+
+    }
+
+    @Override
+    public void update(double pNow) {
+        mCurrentTarget = new TargetingData();
+    }
+
+    @Override
+    public void shutdown(double pNow) {
+
+    }
+
+    public enum LedMode {
+        NO_CHANGE,
+        LED_OFF,
+        LED_BLINK,
+        LED_ON;
+    }
+
+    public enum Stream {
+        STANDARD,
+        PIP_MAIN,
+        PIP_SECONDARY;
+    }
+
+    public void setCamMode(boolean pMode) {
+        mTable.getEntry("camMode").setBoolean(pMode);
+    }
+
+    public void setLedMode(LedMode pMode) {
+        mTable.getEntry("ledMode").setNumber(pMode.ordinal());
+    }
+
+    public void setPipeline(int pipeline) {
+        mTable.getEntry("pipeline").setNumber(pipeline);
+    }
+
+    public void setSnapshot(boolean snapshot) {
+        mTable.getEntry("snapshot").setBoolean(snapshot);
+    }
+
+    public void setStream(Stream stream) { 
+        mTable.getEntry("stream").setNumber(stream.ordinal());
+    }
+
+    
+
```

Comment:
Should use ```Codex<ETargetingData>```, where ```enum ETargetingData implements CodexOf<Double>```.

This allows us to graph these values using existing shuffleboard stuff, including timestamps, row id's, etc.

Reply:
I'm not seeing any reference to ETargetingData in Limelight. Where in the code do you want this change?